src/_guides/language/analysis-options.md

```diff
@@ -173,10 +173,10 @@ You can use the flags together or separately; both default to `true`.
   includes an implicit downcast that would be caught by this flag:
 
 {:.fails-sa}
-<?code-excerpt "analysis/lib/assignment.dart (implicit-downcast)"?>
-{% prettify dart %}
+<?code-excerpt "analysis/lib/assignment.dart (implicit-downcast)" replace="/(s = )(o)/$1[!$2!]/g"?>
+{% prettify dart class="linenums:10 analyzer" %}
```

Comment:
Do we really need linenums here? It's not referred to in the output. (Should it be?)

Reply:
Ok, I've dropped the linenums.

Reply:
Committed. Staging server updated.